lib/js/components/Card/Card.vue

```diff
@@ -14,9 +14,10 @@
 @import '../../../styles/settings/spacings';
 @import '../../../styles/settings/shadows';
 @import '../../../styles/settings/radiuses';
+@import '../../../styles/settings/colors/tokens';
 
 .card {
-	background-color: $color-total-white;
+	background-color: $color-default-background;
```

Comment:
`@import '../../../styles/settings/colors';` is not used now. Please delete it here and in all other components